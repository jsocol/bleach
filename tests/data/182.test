<div id="116"><div id="x">x</div>
<xml:namespace prefix="t">
<import namespace="t" implementation="#default#time2">
<t:set attributeName="innerHTML" targetElement="x" to="<img\u000Bsrc=x\u000Bonerror\u000B=alert(116)>">//["'`-->]]>]</div>
