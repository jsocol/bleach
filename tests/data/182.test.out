<div id="116"><div id="x">x</div>


//["'`--&gt;]]&gt;]</div>
